boards/shared/Inc/App/App_SharedMacros.h

```diff
@@ -13,8 +13,7 @@
 #define __BASENAME__(path) \
     (__builtin_strrchr(path, '/') ? __builtin_strrchr(path, '/') + 1 : path)
 
-#define min(x, y) (((x) < (y)) ? (x) : (y))
-
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))
```

Comment:
There was a name conflict with `min` when running for x86. Not sure why this started now, but I suspect this is a windows thing.